sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/FlexSquidSensorTest.java

```diff
@@ -110,41 +111,46 @@ public void analyse() throws FileNotFoundException {
     assertThat(tester.highlightingTypeAt(componentKey, 2, 0)).containsOnly(TypeOfText.COMMENT);
   }
 
-  private DefaultInputFile inputFile(String fileName) throws FileNotFoundException {
-    return new DefaultInputFile("key", fileName)
-        .setType(InputFile.Type.MAIN)
-        .setLanguage(Flex.KEY)
-        .initMetadata(new FileMetadata().readMetadata(new FileReader(new File(TEST_DIR, fileName))));
+  private DefaultInputFile inputFile(String fileName) throws IOException {
+    File file = new File(TEST_DIR, fileName);
+    String content = new String(Files.readAllBytes(file.toPath()), Charset.defaultCharset());
+
+    return TestInputFileBuilder.create("key", fileName)
+      .setModuleBaseDir(Paths.get(TEST_DIR.getAbsolutePath()))
+      .setType(InputFile.Type.MAIN)
+      .setLanguage(Flex.KEY)
+      .setCharset(Charset.defaultCharset())
```

Comment:
Same thing about the default charset.

Reply:
Moreover, I'm not sure we have to specify a charset here.

Reply:
Ok for charset. It is necessary, otherwise when `inputFile.contents()` is called, it raises a `NullPointerException`